core/rule/src/main/java/com/buschmais/jqassistant/core/rule/api/model/Report.java

```diff
@@ -16,6 +16,8 @@ public class Report {
 
     private String primaryColumn;
 
+    private String keyColumns;
```

Comment:
I'd prefer a `List<String> keyColumns` here, which means that parsing the keyColumns attribute must happen in the Ruleparser plugins.